### PR TITLE
Visual pursuit

### LINKED ISSUE #12: Suggestions for visual pursuit
```
ID: NBO:0000446
Preferred Name : visual pursuit
Definitions: Selectively track a moving object. [NBO:GVG]


1. Definition needs to include mention of vision.  Current term applies to bats 
using echolocation.
2. Needs to be subclass of visual behavior
3. Should not be a subclass of eye movement - as not all animals that can 
visually track can move their eyes.

General question: 
This is under body part movement - should visual pursuit be treated more 
generally, encompassing tracking via whole body movement.


```

Original issue reported on code.google.com by `dosu...@gmail.com` on 24 Feb 2013 at 8:48

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #12

### CODE REVIEW

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -5419,13 +5421,15 @@ SubClassOf(obo:NBO_0000445 obo:NBO_0000001)
 
 # Class: obo:NBO_0000446 (visual pursuit)
 
-AnnotationAssertion(Annotation(oboInOwl:source "NBO:GVG") obo:IAO_0000115 obo:NBO_0000446 "Selectively track a moving object.")
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000446 "Selective tracking that follows a moving object by eye.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:5424]: Missing source

File: src/ontology/nbo-edit.owl [hunk 2]
```diff
@@ -5419,13 +5421,15 @@ SubClassOf(obo:NBO_0000445 obo:NBO_0000001)
...
+AnnotationAssertion(dc:contributor obo:NBO_0000446 <https://orcid.org/ 0000-0002-4277-2151>)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:5425]:
```suggestion
AnnotationAssertion(dc:contributor obo:NBO_0000446 <https://orcid.org/0000-0002-4277-2151>)
```

File: src/ontology/nbo-edit.owl [hunk 3]
```diff
@@ -5419,13 +5421,15 @@ SubClassOf(obo:NBO_0000445 obo:NBO_0000001)
...
 AnnotationAssertion(dc:date obo:NBO_0000446 "2011-04-08T12:25:53Z")
 AnnotationAssertion(oboInOwl:created_by obo:NBO_0000446 "George Gkoutos")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000446 "behavior_ontology")
+AnnotationAssertion(Annotation(oboInOwl:source "NBO:GVG") oboInOwl:remark obo:NBO_0000446 "Original definition: Selectively track a moving object.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:5429]:
```suggestion
AnnotationAssertion(rdfs:comment obo:NBO_0000446 "Original definition: Selectively track a moving object (Source: NBO:GVG).")
```

Comment (matentzn) [reply src/ontology/nbo-edit.owl:5429]: We do not add annotations on comments.

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/nbo-edit.owl (hunk 1 above)
Comment (DitchingIt) [reply src/ontology/nbo-edit.owl:5424]: This is just an intermediate solution to an old issue - we are not yet ready to reorganise the NBO, which will involve detailed academic sourcing.

**Review by matentzn — APPROVED** [review]

**Pull request merged**